### ISSUE TITLE
Accessibility errors - File path : './src/fr/index.md'

### ISSUE THREAD

**Issue by damiengomez** [Orange-OpenSource/a11y-guidelines#899]
- [x] svg-accessible-name-missing — Visible SVG missing an accessible name => Addressed by #931 

`<kbd class="DocSearch-Button-Key"><svg width="15" height="15" class="DocSearch-Control-Key-Icon"><path d="M4.505 4.496h2M5.505 5.496v5M8.216 4.496l.055 5.993M10 7.5c.333.333.5.667.5 1v2M12.326 4.5v5.996M8.384 4.496c1.674 0 2.116 0 2.116 1.5s-.442 1.5-2.116 1.5M3.205 9.303c-.09.448-.277 1.21-1.241 1.203C1 10.5.5 9.513.5 8V7c0-1.57.5-2.5 1.464-2.494.964.006 1.134.598 1.24 1.342M12.553 10.5h1.953" stroke-width="1.2" stroke="currentColor" fill="none" stroke-linecap="square"></path></svg></kbd>`

- [x] Landmark-name-required — Navigation tag on the footer does not expose an accessible name while multiple navigation landmarks are present. => Addressed by #930 

`<nav class="py-2" role="navigation"></nav>`

- [x] zoom-200 — languages choice disapear with a 200% zoom. (it came back in the burger menu at 250% zoom) => Addressed by #932 

- [x] status-message-accessible-text — Visible status message does not provide accessible text => Addressed by #933 
Use role="status" or aria-live="polite" for the messages "Google Tag Manager
autorisé - Ce service a déposé 1 cookie." in the cookies modal

**Comment by MewenLeHo** [comment]
#930, #931, #932 and #933 were all validated and merged.
All problems resolved.
